src/views/Dashboard.vue

```diff
@@ -300,6 +300,7 @@ export default {
         'isAdminFrozen',
         'isBenBcCccUlc',
         'isFirm',
+        'isFutureEffectiveAmalgamation',
```

Comment:
Duplicate reference. To keep things sorted, please delete line 299.

Reply:
Thanks for catching this Sev.